docker/bff-wallet-service/nginx.conf

```diff
@@ -6,7 +6,7 @@ http {
         listen 4242;
 
 
-        location /v1/mint/credit {
+        location /v1/mint/quote/credit {
```

Comment:
Hm, doesn't this trigger a change in `quote-client`? :thinking:

Because e.g. enquire would then go to `/v1/mint/quote/credit/quote` :thinking: